go/chat/unfurl/scrape_generic_scoring.go

```diff
@@ -98,6 +100,40 @@ func getHrefAndContentAttr(e *colly.HTMLElement) []string {
 	return append(getHrefAttr(e), getContentAttr(e)...)
 }
 
+func getOpenGraphVideo(e *colly.HTMLElement) []string {
+	sibs := e.DOM.Siblings()
+	size := sibs.Size()
+	url := e.Attr("content")
+	if len(url) == 0 {
+		return nil
+	}
+	mimeType := "video/mp4"
+	var height, width *int
+	for i := 0; i < size; i++ {
```

Comment:
why can't you query the siblings directly with a selector?

Reply:
I just wanted to get all video information in one place, and not have to have some follow up code to see if I got everything.